euphony/src/test/java/euphony/lib/ui/viewmodel/TxRxCheckerViewModelTest.kt

```diff
@@ -0,0 +1,122 @@
+package euphony.lib.ui.viewmodel
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import co.euphony.rx.AcousticSensor
+import co.euphony.rx.EuRxManager
+import co.euphony.tx.EuTxManager
+import co.euphony.ui.viewmodel.TxRxCheckerViewModel
+import kotlinx.coroutines.DelicateCoroutinesApi
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.newSingleThreadContext
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.setMain
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.Mockito.`when`
+import org.mockito.Mockito.verify
+import org.mockito.junit.MockitoJUnitRunner
+
+@RunWith(MockitoJUnitRunner::class)
+@OptIn(DelicateCoroutinesApi::class)
+class TxRxCheckerViewModelTest {
+
+    @get:Rule
+    val instantExecutorRule = InstantTaskExecutorRule()
+
+    val mainThreadSurrogate = newSingleThreadContext(TxRxCheckerViewModelTest::class.java.simpleName)
+
+    private lateinit var viewModel: TxRxCheckerViewModel
+
+    @Mock
+    private lateinit var txManager: EuTxManager
+
+    @Mock
+    private lateinit var rxManager: EuRxManager
+
+
+    @Before
+    fun setup() {
+        viewModel = TxRxCheckerViewModel(txManager, rxManager)
+        Dispatchers.setMain(mainThreadSurrogate)
+    }
+
+    @After
+    fun tearDown() {
+        Dispatchers.resetMain()
+    }
+
+    @Test
+    fun `if start, livedata should be initialized and Euphony Tx & Rx should work`() {
```

Comment:
I really like test functions' name :)
I didn't know express like this using `` 👍